### PR TITLE
Ship traditional search geek scorers with quepid

### PR DESCRIPTION
**Pull request by epugh**
In our production app.quepid.com you have access to NDCG@10 Scorer because we created it.

Download Quepid and run it, or working in Dev or Staging, and no NDCG@10 because we only seed the old `v1` scorer.

## Description
Introduce more scorers that are read from `./db/scorers/` directory.  These are loaded via `rake db:seed`, and it checks if the scorer already exists or not.  

We may need to rethink the "Default Scorer" is the latest scorer as we add more, or update it periodically to be "the latest".  Or get folks to explicitly pick the Default Scorer to use during setup or something.

## Motivation and Context

This was sparked by conversation with @janhoy, and created the #91 issue.

## How Has This Been Tested?
Manually.

## Types of changes

- [] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: app/assets/javascripts/filters/scoreDisplay.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ angular.module('QuepidApp')
         if (score === '?') {
           return '?';
         } else if ( angular.isNumber(score) ) {
-          return $filter('number')(score, 0);
+          return $filter('number')(score, 2);
```
Comment (epugh) [inline app/assets/javascripts/filters/scoreDisplay.js]: Can you walk me through this change?  I thought we dealt with this change of moving from 0 to 2 in a previous commit that we shipped?

Comment (nathancday) [reply app/assets/javascripts/filters/scoreDisplay.js:11]: I don't think we did. At one point we thought about storing this information in the db's scorer table, but were unable to consistently retrieve it after the initial app load.

**Comment by o19s-admin** [comment]
Cool.

**Comment by janhoy** [comment]
What's left to do here?

**Comment by epugh** [comment]
Tomorrow (April 3rd) I will have some time to give this a good work out and hopefully merge.

**Comment by epugh** [comment]
I'm thinking we need to update the https://github.com/o19s/quepid/wiki/How-Scoring-Works-in-Quepid page and maybe have a link from the Scorers UI to that wiki page.   Need to explain to our audience, who are primarily *new* to relevance, what AP@5 is, and all the other scorers...

**Comment by epugh** [comment]
@nathancday got this branch running, and played around with the scroers etc.  Let's chat next week, as I do have some concerns about this branch for the vast majority of new users..   Great to see the new scorers however!

**Comment by epugh** [comment]
Okay, after discussion with @softwaredoug going to go ahead and pull the trigger on merging this now.  Yes, we still have to get the graph vertical axis working.

**Pull request merged**